models/contracts/chains/contracts_base_contract_creator_project_iterated_creators.sql

```diff
@@ -0,0 +1,15 @@
+ {{
+  config(
+        schema = 'contracts_base',
+        alias = 'contract_creator_project_iterated_creators',
+        materialized ='incremental',
```

Comment:
these spells are incremental, yet the macro has no incremental logic. what is the reason behind that?

Reply:
copy and paste issue, making a materialized table (unless it should be something else)?